tests/test_sample.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os
+import json
+
+with open("tests/to_validate.json", "r", encoding="utf-8") as file:
```

Comment:
Would it make sense to move these files to `tests/assets` or `tests/validation_files` or similar? That way we separate what are actual tests from test assets.

Reply:
Yes, you are right! This is a best practice. Done!  [5272383](https://github.com/ivica-k/terraform-validate-explorer/pull/11/commits/5272383a7e7f3214663fbaf85edcfa29ec01cb45)

Reply:
Test on macOS and python 12 !
<img width="1135" alt="image" src="https://github.com/user-attachments/assets/73bb119d-ff63-44f4-a3cf-e685277111c7" />